bgc_data/bgc_zoop_raw.dataresource.yaml

```diff
@@ -0,0 +1,49 @@
+profile: tabular-data-resource
+name: bgc_zoop_raw
+path: https://www.cmar.csiro.au/geoserver/imos/wfs?service=wfs&version=1.1.0&request=GetFeature&outputformat=csv&typename=imos%3Abgc_zoop_raw
+title: bgc_zoop_raw
+description: "A table of all BGC zooplankton change log data"
```

Comment:
```suggestion
description: "A table of all BGC zooplankton abundance data"
```

Reply:
Hi Marty - latest commit should fix this.